### PR TITLE
Navigation 

### PR DESCRIPTION
**Pull request by MirekSobota**
This is still not finished yet, but you can use it to create your components :)

### CODE REVIEW

**Comment by PawelOwiesek** [comment]
Mirek is the gap size ok? I have 12px in Figma in place of Your 16.

File: src/common/Navigation/Header/styled.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    flex-wrap: nowrap;
+    gap: 16px;
```
Comment (MirekSobota) [inline src/common/Navigation/Header/styled.js:7]: @PawelOwiesek you are correct, i will change it to 12px.  You are very attentive, nice!

**Comment by DorotaZielinska** [comment]
Most of it is ok.

**Comment by MirekSobota** [comment]
I will still working on it, next branch will be much more better! Thanks for your reviev!

**Comment by DorotaZielinska** [comment]
Is fine

**Review by DorotaZielinska — APPROVED** [review]

**Review by PawelOwiesek — COMMENTED** [review]
Looks good for me

**Review by PawelOwiesek — COMMENTED** [review]
Its ready to marge

**Review by PawelOwiesek — APPROVED** [review]

**Pull request merged**